packages/components/src/InlineLabel/InlineLabel.css

```diff
@@ -1,18 +1,32 @@
 .inlineLabel {
   display: inline-flex;
-  border-radius: var(--radius-large);
 }
 
 .base {
   padding: calc(var(--space-smaller) * 1.5) calc(var(--space-small) * 1.25);
+  border-radius: var(--radius-large);
+
+  span {
+    line-height: 0.75rem;
+  }
```

Comment:
When bootstrapping Atlantis (`npm run lerna:clean && npm ci`) this warning comes up:

![Screenshot 2024-03-25 at 12 51 46 PM](https://github.com/GetJobber/atlantis/assets/104797704/6d6cbf84-7964-4fd0-baf5-1e8e5f465c1c)


So instead of the nested span, we should target the spans this way:

![Screenshot 2024-03-25 at 12 41 03 PM](https://github.com/GetJobber/atlantis/assets/104797704/53739a5d-ae04-4665-86e0-c040677aa5a7)